mlrun/serving/remote.py

```diff
@@ -527,3 +527,86 @@ def post_init(self, mode="sync", **kwargs):
         self.fill_placeholders = self.fill_placeholders or False
         self.rundb = mlrun.get_run_db()
         self.url = self.rundb.get_base_api_url(self.path)
+
+
+class RemoteFunctionStep(RemoteStep):
+    """
+    Graph step implementation for invoking functions remotely.
+    :param fn: Either an mlrun.runtimes.RemoteRuntime object or
+    a string URI in the form 'function_name' or 'project_name/function_name'
+    :param project_name: Project containing the function.
```

Comment:
I'm not sure if it's a good idea to have two options to specify the project (in the `fn` or the `project` params) - which one wins if they are both specified?